packages/ag-charts-enterprise/tsconfig.lib.json

```diff
@@ -3,6 +3,10 @@
   "compilerOptions": {
     "types": ["node"]
   },
-  "include": ["src/**/*.ts"],
+  "include": [
+    "src/**/*.ts",
+    "../ag-charts-community/src/chart/series/topology/mercatorScale.ts",
+    "../ag-charts-community/src/chart/series/topology/lonLatBbox.ts"
```

Comment:
This looks wrong, can you export these via the `main.ts` in `ag-charts-community` instead?